packages/govern-subgraph/schema.graphql

```diff
@@ -16,28 +16,18 @@ type Govern @entity {
   id: ID!
   address: Bytes!
   metadata: Bytes
-  games: [OptimisticGame!] @derivedFrom(field: "executor")
-  executions: [Execution!]
+  correspondingEntries: [RegistryEntry!] @derivedFrom(field: "executor")
```

Comment:
Still not happy with this name, but going with it for now. Suggestions are welcome. :D

Reply:
I think games were pretty appealing given that DAOs are going to be participating on this optimistic games. What is the reason to instead linked to entities on a registry?

Reply:
DAOs might be participating in optimistic games, but they're _not_ optimistic games by themselves. I think it's easier to think of these entities as entries in the registry, mainly because it's what they are in the contracts.

Reply:
Btw @bpierre Yep, def agree about the naming change. What about sticking to `registryEntry` and `registryEntries`? I think it sounds natural, even if it's 100% aligned with the method's verb. :)

Reply:
Sounds good to me! @0xGabi what do you think?

Reply:
Yes, sound good